tests/foreman/api/test_operatingsystem.py

```diff
@@ -63,55 +63,50 @@ def test_verify_bugzilla_1114640(self):
         os_param = entities.OperatingSystemParameter(
             name=name, operatingsystem=1, value=value
         ).create()
-        self.assertEqual(os_param.name, name)
-        self.assertEqual(os_param.value, value)
+        assert os_param.name == name
+        assert os_param.value == value
 
 
-class OperatingSystemTestCase(APITestCase):
+class TestOperatingSystem:
```

Comment:
I think class is not required.

Reply:
There are currently two different classes in the same module. I'm keeping the classes in place for now, so that the tests are kept logically distinct.

Reply:
yes, you are right, but the same thing we can also achieve by putting the test details inside the test case doc.  This is just my suggestion, I don't think the class is actually required here.